includes/class-core.php

```diff
@@ -434,8 +434,7 @@ public function append_footer_script_for_collapsed_flexible_content_layouts() {
 		<script>
 			(function( $ ) {
 				$( document ).ready( function( ) {
-					$('.acf-flexible-content .layout').addClass('-collapsed');
-					$('.acf-flexible-content .acf-postbox').addClass('closed');
+					$('.acf-flexible-content .layout[data-layout!="text"]').addClass('-collapsed');
```

Comment:
Lurer på om det kanskje er bedre å bruke `.layout:not([data-layout="text"])`?

Reply:
Googla litt og er enig. Kommer en commit straks.